topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/workflows/PeptideML.ga

```diff
@@ -0,0 +1,734 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
+    "name": "PeptideML",
```

Comment:
Could you add: 
`"tags": ["proteomics"]`
`"annotation": "<title of tutorial>"`

That will fix the linting error. (same for the other workflow)

Reply:
Thank you for the help, I updated WF files.